src/diffusers/models/transformers/transformer_z_image.py

```diff
@@ -658,4 +630,7 @@ def forward(
         unified = list(unified.unbind(dim=0))
         x = self.unpatchify(unified, x_size, patch_size, f_patch_size)
 
-        return x, {}
+        if not return_dict:
+            return (x,)
+
+        return Transformer2DModelOutput(sample=x)
```

Comment:
Should be a very safe change?

Reply:
yes indeed, is this the only actual change in this file? (others seems just format changes)